### DISCUSSION TITLE
Wazuh account creation

### DISCUSSION THREAD

**Discussion by b4b857f6ee** [Security-Onion-Solutions/securityonion#7112] [Unsupported Versions] [answered]
Hello,

I'm on 2.3.100

I'm trying to create a user only in commande line using : so-wazuh-user-add newuser
I can find how to do it. I need do to it without installing extra package like expect.
Any idea?
I don't find how to fill the password automaticaly when asking :/

Thank you for your help :)

**Comment by dougburks** [comment, marked as answer]
Take a look at `so-wazuh-user-add` and you will see that it is just a simple wrapper that runs the following:
```
docker exec -it so-wazuh /usr/bin/node /var/ossec/api/configuration/auth/htpasswd /var/ossec/api/configuration/auth/user $1
```

So instead of running `so-wazuh-user-add`, you could run the following and it will give you options that you can use for scripting:
```
docker exec -it so-wazuh /usr/bin/node /var/ossec/api/configuration/auth/htpasswd /var/ossec/api/configuration/auth/user --help
```

**Reply by b4b857f6ee** [reply]
YESSSSS it's work, i was testing with the command but i didn't notice the "-b" option.
Love you, thank you :D!!!!
